activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -58,6 +58,15 @@ def custom_primary_key?
         options[:primary_key] != default_primary_key
       end
 
+      def validate
+        options.fetch(:validate, true)
+      end
```

Comment:
There are other predicate methods that do not return boolean (e.g. `connection.in_use?`, `connection.strict_mode?`.
I think it should be named according to how it is used.